main.tf

```diff
@@ -66,7 +66,7 @@ data "aws_iam_policy_document" "ec2_cloudwatch" {
 }
 
 resource "aws_iam_role" "ec2_cloudwatch" {
-  name = "${module.label.id}"
+  name = "${module.label.id}_cloudwatch_agent"
```

Comment:
add `attributes = ["cloudwatch-agent"]` to the `label module, and then just use 

```
name = "${module.label.id}"
```

Reply:
Awesome, thanks!